soh/soh/Enhancements/cosmetics/CosmeticsEditor.cpp

```diff
@@ -1035,6 +1037,15 @@ void ApplyOrResetCustomGfxPatches(bool manualChange = true) {
         PATCH_GFX(gNintendo64LogoDL,                              "Title_N64LogoYellow2",       n64LogoYellow.changedCvar,           82, gsDPSetEnvColor(color.r, color.g, color.b, 128));
     }
 
+    // Greg Bridge
+    if (Randomizer_GetSettingValue(RSK_RAINBOW_BRIDGE) == RO_BRIDGE_GREG) {
```

Comment:
Main drawback of putting it here is this will run either the unpatch or the patch every frame which isn't super great. Not sure I can think of a better way to handle it right now though

Reply:
yeah, i spent a while trying to think of a good spot for it, i kind of figured here works because it's where we're doing a ton of other patching